examples/sample-layers/plot-layer/grid-vertex.glsl

```diff
@@ -25,6 +25,8 @@ attribute vec3 normals;
 attribute vec2 instancePositions;
 attribute vec3 instanceNormals;
 
+uniform mat4 modelMatrix;
```

Comment:
I wonder if we need to expose `modelMatrix` separately. See comment below on perf impact.